File: flink_train_scala/src/main/scala/com/learn/flink_scala/DataSetDataSourceApp.scala
package com.learn.flink_scala

import org.apache.flink.api.scala.ExecutionEnvironment

/**
  * Created by kimvra on 2019-05-09
  */
object DataSetDataSourceApp {
  def main(args: Array[String]): Unit = {
    val env = ExecutionEnvironment.getExecutionEnvironment
    fromCollection(env)
  }

  def fromCollection(environment: ExecutionEnvironment) = {
    val data = 1 to 10
    import org.apache.flink.api.scala._
    environment.fromCollection(data).print()
  }
}

File: flink_train_java/src/main/java/com/learn/flink_java/demo1/DataSetDataSourceApp.java
package com.learn.flink_java.demo1;

import org.apache.flink.api.java.ExecutionEnvironment;

import java.util.Arrays;

/**
 * Created by kimvra on 2019-05-09
 */
public class DataSetDataSourceApp {
    public static void main(String[] args) throws Exception{
        ExecutionEnvironment env = ExecutionEnvironment.getExecutionEnvironment();
        fromCollection(env);
    }

    private static void fromCollection(ExecutionEnvironment env) throws Exception{
        env.fromCollection(Arrays.asList(1,2,3)).print();

    }
}
